src/serial/SerialArea.tsx

```diff
@@ -14,6 +14,7 @@ import XTerm from "./XTerm";
 interface SerialAreaProps extends BoxProps {
   compact?: boolean;
   onSizeChange: (size: "compact" | "open") => void;
+  showTerminal?: boolean;
```

Comment:
We should decide whether we need this now. It might be better for sim serial not to come and go? But then I guess it'll need a disabled state. Worth some discussion but can be after this PR as its location is very much not final too.